Felienne/hedy
# Ace now highlights keywords from other languages

**Description**

- highlights the keywords correctly in the online ace instance, should the option be turned on.
- contains syntaxModesRules-en and -nl, templates for the creation of modes for the highlighter in ace/modes/level1en.js as opposed to just ace/modes/level1.js
- syntaxRules.ts now contains exported rules shared by each individual highlighter, to prevent duplicate code errors.
- The original syntaxModesRules.ts has been changed to syntaxModesRules-en.ts, which serves the same function as the original
- slight changes in app.ts and index.ts to allow for these different highlighting rules to actually be used (importing the syntaxModesRules-'lang'.ts files and calling the correct ace/mode file), also includes an enable to turn this on or off. It is currently off by default.

**Fix for**

Closes #909 

**How to test**

Set **local_keywords_highlights_enabled** in app.ts to true and rebuild the appbundle, afterwards run the website. You should now see that the keywords highlighted for the current language (currently just dutch and english) are highlighted appropriately according to the language's keywords as given in 'lang'.yaml
So print and echo should both be highlighted for level 1, forward should only highlight for english, vooruit should only highlight in dutch, etc.
